bazel/coverage/fuzz_coverage_wrapper.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -ex
+
+TEST_BINARY=$1
+shift
+
+mkdir -p fuzz_corpus/seed_corpus
```

Comment:
What happens to fuzz_corpus after the run is over? Will it need to be manually cleaned up?

Reply:
No it is under bazel sandbox so will be cleaned up by bazel.